Jenkinsfile

```diff
@@ -21,7 +21,9 @@
         steps {
           container('nodejs') {
             sh "npm install"
-            sh "npm run build"
+            //sh "npm run build"
+            sh "npm run e2e"
```

Comment:
For running e2e you need to set some environment vars to work properly in Jenkins, I assume:
```
API_HOST=""
OAUTH_HOST=""
E2E_HOST="http://localhost"
E2E_PORT="4100"
E2E_USERNAME=""
E2E_PASSWORD=""
E2E_UNAUTHORIZED_USER=""
E2E_UNAUTHORIZED_USER_PASSWORD=""
E2E_FAIL_FAST="true"
BROWSER_RUN="false"
SAVE_SCREENSHOT="true"
SCREENSHOT_URL=""
SCREENSHOT_USERNAME=""
SCREENSHOT_PASSWORD=""
```

and obviously you need a BE for the FE to work with. Do you have it?
@salaboy ?

Reply:
Hey @popovicsandras, it is in process and I’m added env already but it is still doesn’t work on nodejs builder.

Reply:
@popovicsandras from https://github.com/Activiti/activiti-modeling-app/blob/master/.travis.yml is not clear how backend is running can you please advice?

Reply:
@almerico, About how backend is running, I'm afraid I can't tell you too much. We are using our staging environments to test against in Enterprise, either aps2dev or aps2staging.
I would expect to have those staging environments for Community also, like acm2dev or acm2staging?